packages/plebeia/plebeia.2.0.0/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+synopsis: "Functional storage using Merkle Patricia tree"
+maintainer: "jun.furuse@dailambda.jp"
+authors: "DaiLambda, Inc"
+license: "MIT"
+homepage: "https://gitlab.com/dailambda/plebeia/"
+bug-reports: "https://gitlab.com/dailambda/plebeia/-/issues"
+depends: [
+  "dune" {>= "3.2.0"}
+  "bigstring" {= "0.3"}
+  "hacl-star" {= "0.4.5"}
+  "blake3" {= "0.2"}
+  "cstruct" {= "6.1.0"}
+  "stdint" {= "0.7.0"}
+  "hex" {= "1.5.0"}
+  "data-encoding" {= "0.5.3"}
+  "ptime" {= "1.0.0"}
+  "alcotest" {= "1.5.0"}
```

Comment:
Your constraints seem to be extremely strict and will cause a lot of problems for your users.
```suggestion
  "bigstring" {>= "0.3"}
  "hacl-star" {>= "0.4.5"}
  "blake3" {>= "0.2"}
  "cstruct" {>= "6.1.0"}
  "stdint" {>= "0.7.0"}
  "hex" {>= "1.5.0"}
  "data-encoding" {>= "0.5.3"}
  "ptime" {>= "1.0.0"}
  "alcotest" {>= "1.5.0" & with-test}
```
Also some of your tests dependencies don’t seem to be marked as is